mysys/crc32/crc32_arm64.c

```diff
@@ -299,3 +300,35 @@ uint32_t crc32c_aarch64(uint32_t crc, const unsigned char *buffer, uint64_t len)
 
 	return (~crc);
```

Comment:
To align with my suggestion for `my_sys.h`, please change this function definition to
```c
ha_checksum crc32c_aarch64(ha_checksum crc, const void *buffer, size_t len)
{
```

Reply:
I left it out now that we are not declaring it in my_sys.h and we will do ut0crc32 changes when we revamp crc32c.